spec/lib/email/receiver_spec.rb

```diff
@@ -879,7 +879,16 @@ def create_post_reply_key(value)
 
     it "cap the number of staged users created per email" do
       SiteSetting.maximum_staged_users_per_email = 1
-      expect { process(:cc) }.to change(Topic, :count)
+      expect { process(:cc) }.to change(Topic, :count).and change(User, :count).by(1)
```

Comment:
Do we want to test the number of topics created?

```suggestion
      expect { process(:cc) }.to change(Topic, :count).by(1).and change(User, :count).by(1)
```